demo-app/.eslintrc.json

```diff
@@ -0,0 +1,51 @@
+{
+  "root": true,
+  "ignorePatterns": [
+    "projects/**/*"
+  ],
+  "overrides": [
+    {
+      "files": [
+        "*.ts"
+      ],
+      "parserOptions": {
+        "project": [
+          "tsconfig.json",
+          "e2e/tsconfig.json"
```

Comment:
There's nothing at this path. Please, remove it if you don't have any reasons to keep it here, otherwise, it's the cause of the error.

Reply:
Good point, it was automatically added by `@angular-eslint/schematics`. Thank you for looking into it!